models/model_draft.md

```diff
@@ -0,0 +1,48 @@
+# Models
+
+Proposed model for snacks and orders.
+
+## Snacks Collection
+
+```json
+{
+    _id: <ObjectId_Snack1>,
+    snackName: "Tortilla Chips",
+    detail: 
+        {
+            flavor: "Cheese",
```

Comment:
Didn't realize it earlier - as we're serving business in the UK, I suggest we use British English also in our domain language
```suggestion
            flavour: "Cheese",
```